easybuild/easyblocks/i/ipp.py

```diff
@@ -101,7 +120,7 @@ def sanity_check_step(self):
 
         custom_paths = {
             'files': [
-                os.path.join('ipp', 'lib', 'intel64', 'libipp%s') % y for x in ipp_libs
+                os.path.join('ipp', self.version, 'lib', 'intel64', 'libipp%s') % y for x in ipp_libs
```

Comment:
are you sure this is correct for <= 2021.2.0 as well? It looks like an if statement is missing here.